.github/CODEOWNERS

```
@@ -77,6 +77,7 @@
 /core/domain/classifier*.py @oppia/ml
 /proto/ @oppia/ml
 /core/templates/domain/classifier/ @oppia/ml
+/core/templates/services/classifier-data-backend-api.service.ts @oppia/ml
```

```suggestion
/core/templates/services/classifier-data-backend-api.service*.ts @oppia/ml
```
(To codeown test files)

---

Done

---

Done